README.md

```diff
@@ -3,3 +3,42 @@
 Pulumi components carefully crafted by Nuage
 
 TODO: use [the official repo](https://github.com/pulumi/pulumi-component-provider-py-boilerplate) to build a Pulumi Component Provider
+
+# Install the Pulumi Plugin
+
+To use this resource, you need to install Pulumi plugin first (independent from the development language).
+
+1. Download the latest release from https://github.com/nuage-studio/pulumi-nuage/releases/
+2. Install the .tar.gz package using Pulumi
+```
+pulumi plugin install resource nuage 0.0.1 --file pulumi-resource-nuage-v0.0.1-darwin-amd64.tar.gz
+```
+
+You may also automate this process via Taskfile. Here is an example:
+```
+version: "3"
+
+tasks:
+  pulumi_install:
+    desc: Downloads binary
+    env:
+      FILE_NAME: "pulumi-resource-nuage-v0.0.1-darwin-amd64.tar.gz"
+    cmds:
+      - curl -L -o $FILE_NAME https://github.com/nuage-studio/pulumi-nuage/releases/download/0.0.1/${FILE_NAME}
+      - pulumi plugin install resource nuage 0.0.1 --file $FILE_NAME
+      - rm $FILE_NAME
+```
+
+# Setup Instructions for Python SDK
+
+* You can add following line under the `[tool.poetry.dependencies]`, to use Python package.
+```
+pulumi-nuage = { git = "https://github.com/nuage-studio/pulumi-nuage.git", subdirectory = "sdk/python" }
+```
+* Then you need to run `poetry install`
```

Comment:
Please use this code snipper instead:
```bash
poetry add "https://github.com/myorg/mypackage_with_subdirs.git#subdirectory=subdir"
```

It'd a direct bash command instead of writing in the TOML file then poetry update, so it's a bit more elegant

Also you could indicate this documentation link to explain that we're using a subdirectory dependency:
https://python-poetry.org/docs/dependency-specification/